src/window/window_wrapper/mod.rs

```diff
@@ -222,7 +222,13 @@ pub fn start_loop(
         .with_decorations(!SETTINGS.get::<CmdLineSettings>().frameless);
 
     #[cfg(target_os = "linux")]
-    let winit_window_builder = winit_window_builder.with_app_id("Neovide".to_string());
+    let winit_window_builder = winit_window_builder
+        .with_app_id(SETTINGS.get::<CmdLineSettings>().wayland_app_id.to_string());
```

Comment:
Try to get rid of this `to_string()` conversion. Make `wayland_app_id` of that type which is expected in with_app_id call.